package.json

```diff
@@ -11,35 +11,35 @@
 	},
 	"types": "main.d.ts",
 	"install-peers": "install-peers",
-	"scripts": {
-		"build": "yarn build-fandom-player && yarn build-stand-alone",
-		"build-fandom-player": "yarn run create-dist && rollup -c && cp package.json dist/ && cd dist && npm set-script prepare ''",
-		"build-stand-alone": "cd stand-alone && yarn && yarn run create-standalone-dist && rollup -c rollup-stand-alone.config.js && cp package.json standalone-dist && npm set-script prepare ''",
-		"deploy-stand-alone": "ts-node --project stand-alone/tsconfig-stand-alone.json stand-alone/deploy/deploy.ts",
-		"watch": "yarn run create-dist && NODE_ENV=development rollup -c --watch",
-		"app": "cd test-jw && yarn && yarn start",
-		"create-dist": "rm -rf dist && mkdir -p dist",
-		"create-standalone-dist": "rm -rf standalone-dist && mkdir -p standalone-dist",
-		"cp:loop": " while true; do cp dist/bundle.esm.js test-jw/src; sleep 10; done",
-		"lint": "yarn eslint src && yarn prettier --check src",
-		"lint:fix": "npm-run-all --continue-on-error --parallel lint:prettier lint:eslint",
-		"lint:prettier:src": "prettier --write src",
-		"lint:prettier:scripts": "prettier --write scripts",
-		"lint:prettier:tests": "prettier --write tests",
-		"lint:prettier": "npm-run-all --continue-on-error --parallel lint:prettier:src",
-		"lint:eslint:src": "eslint --cache --fix src",
-		"lint:eslint:scripts": "eslint --cache --fix scripts",
-		"lint:eslint:tests": "eslint --cache --fix tests",
-		"lint:eslint": "npm-run-all --continue-on-error --parallel lint:eslint:src",
-		"test": "echo 'No Tests yet'",
-		"prepare": "husky install",
-		"pub:patch": "npm version patch && rm -rf dist && yarn pub",
-		"pub:minor": "npm version minor && rm -rf dist && yarn pub",
-		"pub:major": "npm version major && rm -rf dist && yarn pub",
-		"pub": "yarn install && yarn build && cd dist && npm publish --tag beta",
-		"purgeSingleFile": "ts-node --project stand-alone/tsconfig-stand-alone.json stand-alone/deploy/purge/purgeSingleFile.ts",
-		"serveFilesOnBrowser": "npx browser-sync start --server --files \"**/.js\""
-	},
+  "scripts": {
```

Comment:
It seems some out formatter broke the formatting in the package.json 🙈 We used to follow https://github.com/airbnb/javascript#whitespace and there was a way to easily import it to an IDE 🤔

Reply:
Yeah, on my end looks as described in https://github.com/airbnb/javascript#whitespace

Reply:
Weird. Well, it's not a blocker just looked odd.